pkg/arvo/sys/vane/behn.hoon

```diff
@@ -335,7 +335,7 @@
     ~
   ?.  ?=(%timers syd)
     [~ ~]
-  [~ ~ %noun !>((turn (tap:timer-map timers) head))]
+  [~ ~ %noun !>(>(turn (tap:timer-map timers) head)<)]
```

Comment:
Worth noting that the debug dashboard (#2875) makes the opposite change here.

Reply:
Considering a couple things there might need to be updated to match, I think I should just resolve this as a merge conflict on #2875's side.

(Why we ever did, or would want to, produce a tank as a scry result in the first place, idk...)